apps/omg/lib/omg/state.ex

```diff
@@ -222,4 +224,17 @@ defmodule OMG.State do
     :ok = OMG.Bus.broadcast("events", {:preprocess_emit_events, event_triggers})
     :ok = OMG.Bus.direct_local_broadcast("blocks", {:enqueue_block, block})
   end
+
+  @spec init_utxos_from_db(list(InputPointer.Protocol.t())) :: UtxoSet.t()
+  defp init_utxos_from_db(utxo_pos_list) do
+    utxo_pos_list
+    |> Enum.map(&utxo_from_db/1)
+    |> Enum.reject(&(:not_found == &1))
+    |> UtxoSet.init()
+  end
+
+  defp utxo_from_db(input_pointer) do
+    with {:ok, utxo_kv} <- DB.utxo(InputPointer.Protocol.to_db_key(input_pointer)),
```

Comment:
As we talked - we need to handle :not_found. Now they're filtered out in UtxoSet.init

Reply:
maybe a better way of handling this would be to let the developer know what output's you're able to handle
```
case DB.utxo(InputPointer.Protocol.to_db_key(input_pointer)) do
  :not_found -> :not_found
  {:ok, _utxo_kv} = result -> result
end
```
this means that if you get a value back you're not able to handle, you would *fail and crash early*